### PR TITLE
Add example preview to video block.

### LINKED ISSUE #17655: Enhancement: Add video example to the block library
In #17493, we area adding block examples to most blocks.

Whereas all the images are stored on a nonprofit Wikimedia server with an [explicitly permissing hotlinking policy](https://commons.wikimedia.org/wiki/Commons:Reusing_content_outside_Wikimedia/technical), the video needs to be on a CDN, ideally a WordPress.org CDN.

This takes a little bit and in the name of shipping improvements to the other blocks, we're extracting that one out. So when there's time, let's add back this:

65885530-21527980-e39b-11e9-9006-495364ce4bac

Here's the video:

https://make.wordpress.org/design/files/2019/03/block-manager.mp4

### PR DESCRIPTION
**Pull request by mtias**
Closes #17655 

image

### CODE REVIEW

**Comment by mtias** [comment]
cc @jasmussen

**Comment by Soean** [comment]
Internet Explorer 11 doesn't support the webm format, see [caniuse](https://caniuse.com/#feat=webm). Maybe we should us a MP4 file.

**Comment by jasmussen** [comment]
This video is so lovely.

But there are some challenges with it. [Hotlinking files on Wikimedia are explicitly fine](https://commons.wikimedia.org/wiki/Commons:Reusing_content_outside_Wikimedia/technical), and I would suggest that hover previews not working in IE11 is not a blocker because it's not accessibility related, and it's fundamentally necessary for the editor to work. 

However the video is CC licensed 😩 

https://commons.wikimedia.org/wiki/File:Wood_thrush_in_Central_Park_switch_sides_(16510).webm

While we could use it with attribution, in the past we've avoided that because of the challenge of _where do we show that attribution_. 

This one is public domain, though: https://commons.wikimedia.org/wiki/File:Wood_thrush_(Hylocichla_mustelina).webm — it's not as good a video, but perhaps it serves the same purpose?

**Comment by mtias** [comment]
We are just linking to it, not embedding though. We could also just use a poster image since it doesn't seem to autoplay anyways.

**Comment by gziolo** [comment]
Should we land it?

**Comment by mtias** [comment]
Yes, I think so

**Comment by gziolo** [comment]
I'll rebase and merge 😄

**Comment by gziolo** [comment]
I rebased and updated the example to include the caption, this is how it works on Safari:

Screen Shot 2020-12-01 at 17 59 57

I think `autoplay` is disabled in the editor so it doesn't interfere with the writing flow. `controls` is enabled by default. I delete both of them from the example.

Let's decide what we do about this PR given the issues the video creates on the browser support level and in terms of licensing as pointed out by @jasmussen.

**Comment by mtias** [comment]
It's just a link to a wikimedia resource

File: packages/block-library/src/video/index.js [hunk 1]
```diff
@@ -23,6 +23,14 @@ export const settings = {
 	),
 	icon,
 	keywords: [ __( 'movie' ) ],
+	example: {
+		attributes: {
+			src:
+				'https://upload.wikimedia.org/wikipedia/commons/c/ca/Wood_thrush_in_Central_Park_switch_sides_%2816510%29.webm',
+			// translators: Caption accompanying a video of the wood thrush singing, which serves as an example for the Image block.
```
Comment (gziolo) [inline packages/block-library/src/video/index.js:30]:
```suggestion
			// translators: Caption accompanying a video of the wood thrush singing, which serves as an example for the Video block.
```

**Review by gziolo — APPROVED** [review]

**Pull request merged**